sean/class2-2/9012.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+# sys.stdin = open("input.txt", "r")
+
+t = int(input())
+
+for _ in range(t):
+    text = input()
+    stack = []
+    for c in text:
+        if c == "(":
+            stack.append("(")
+        else:
+            if len(stack) == 0:
+                stack.append("(")
+                break
+            stack.pop()
+
+    print("YES" if len(stack) == 0 else "NO")
```

Comment:
```suggestion
    print("YES" if not stack else "NO")
```

stack의 len이 0이라는건 비어있다는 것(False)이잖아요. 그래서 저는 이렇게 쓰는게 조금 더 간결한 것 같더라구요!

Reply:
오 python에서 자료형마다 falsy 값들이 존재하는걸 몰랐네요
조언 감사해요!!